### ISSUE TITLE
Change 'Live Code Analysis' text

### ISSUE THREAD

**Issue by kendrahavens** [dotnet/roslyn#37751]
The 'Live Code Analysis' task center notification is causing confusion as many users believe any errors/delays occurring in Visual Studio are due to this process. This process is actually several low priority background tasks which are often paused when Visual Studio is executing higher priority tasks.

The purpose of the notification was only to acknowledge that Visual Studio often executes a variety of tasks in the background that should not interfere with the user experience, but would be detectable if users are monitoring their Task Manager.

### Current UI
![image](https://user-images.githubusercontent.com/8293542/62505026-d4459300-b7af-11e9-949e-074f9a5b4e06.png)

### Proposed change
* Show when this state is paused and waiting on higher priority tasks
* Change title to be more generic so users don't believe "live analysis" is the culprit by default when experiencing slow UI.
![image](https://user-images.githubusercontent.com/8293542/62504926-7022cf00-b7af-11e9-99d2-164a0d0f0f76.png)

**Comment by chucker** [comment]
This seems like an improvement, but:

> This process is actually several low priority background tasks

I think part of the confusion is the UI makes it look as though it _shows_ the background tasks. This issue sounds like there are background tasks that are being enumerated somewhere, but not surfaced (by name or otherwise) in the UI. Is this a deliberate design decision?

[ Also, this was linked from a developer community issue, which to my understanding was in part about the UI freezing (temporarily). “Background” suggests to me that this shouldn’t happen with those tasks. ]

**Comment by jinujoseph** [comment]
@chucker  idea is to surface the ## of tasks pending to be processed in the queue.  The intension is to bring clarify over since people associate the tasks to be associated with there UI freeze which may not be true.

**Comment by kendrahavens** [comment]
@chucker Yes, I should have called out, the linked bug did result in several fixes that will be available in 16.2, but the issue continues to accumulate votes and comments for unrelated issues because the 'Live Code Analysis' message is present almost all the time.

We considered listing out what background tasks were being executed as they happened, but this led to a flickering UI that we believed did not provide actionable information.

**Comment by chucker** [comment]
Thanks for the clarification!

**Comment by chandmk** [comment]
Users like me are looking at the task bar only because, VS 2019 is not working.

Ex: If I rename a variable in a large file, which is being used elsewhere in the file, I expect red squiggly lines show up in all of its usages.  But in the latest release version of VS2019 (16.2.1 and earlier releases of VS2019) error appear after a minute.  While waiting for the VS to display the usage errors,  user can't help noticing the only animating part in the VS at that time.  Live Analysis progress bar in the left lower corner says "Analyzing the <file.cs>".  Live Analysis progress bar starts as soon as the code change was made.  Once the progress bar stops, red squiggly lines appear.  

I don't understand how are you going to fix the actual problem by just renaming help text in the UI?

**Comment by CyrusNajmabadi** [comment]
@kendrahavens @jinujoseph  I don't think this issue should be closed.  We've addressed a symptom, but not the root causes.  Things like live-analysis *are* runnin very slowly.  for example, the issue @chandmk reported was also reported by me here: https://github.com/dotnet/roslyn/issues/37208

Things are *really* slow in VS.  Noticeably slower than in the past.  And definitely slower than Resharper.  People see the slowness and def blame background analysis.  But that's appropriate.  The slowness they're seeing is the slowness in background analysis (i.e. getting errors in the background).

**Comment by jinujoseph** [comment]
@CyrusNajmabadi this issue is tracking the Text change only.
The performance issue associated is still an active bug looked separately.
